lang/en/assignsubmission_maharaws.php

```diff
@@ -27,22 +27,22 @@
 $string['collectionsby'] = 'Collections by {$a}';
 $string['debug_help'] = 'Debug option to interupt the OAuth SSO login jump so parameters can be inspected';
 $string['debug'] = 'Debug OAuth';
-$string['defaultlockpages_help'] = 'Default setting to use for the "{$a}" setting in new Mahara assignments.';
+$string['defaultlocks_help'] = 'Default setting to use for the "{$a}" setting in new Mahara assignments.';
```

Comment:
It's not usually recommended to change the lang string itself as it will affect possible translations. However, I don't think this plugin is being used widely so that should be ok as it won't affect too many users.

Reply:
I'm fine with keeping the string identifier for this one if that is best.

Reply:
Reverted the change.